tasks/main.yml

```diff
@@ -89,20 +112,13 @@
     mode: 0640
     force: no
 
-- name: Update the system /etc/sudoers file
-  template: >-
-    src=sudoers.j2
-    dest=/etc/sudoers.d/{{ item.username }}
-    mode=0600
-  when: item.active | default(false)
-  with_items: "{{ jumpbox_operators }}"
```

Comment:
You'll want to add a task to clean up these sudoers files on the existing jumpbox. Remember, we're not starting from a clean install.